comms/src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub use peer_manager::{NodeIdentity, PeerManager};
 
 pub mod framing;
 
-mod common;
+pub mod common;
```

Comment:
nit: I'd rather re-export the rate_limit module, not the common module.

```suggestion
mod common;
pub use common::rate_limit;
```